exporters/otlp/otlpmetric/options.go

```diff
@@ -0,0 +1,34 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otlpmetric
+
+import metricsdk "go.opentelemetry.io/otel/sdk/export/metric"
+
+// ExporterOption are setting options passed to an Exporter on creation.
+type ExporterOption func(*config)
```

Comment:
This doesn't match the project configuration pattern: https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#option

Reply:
Updated that to conform the project pattern.